docs/doxygen/doxygen_wrapper.py

```diff
@@ -167,6 +174,35 @@ def _cleanup_dir_by_extension(child_dir: Path, extension: str):
         for future in cleanup_futures:
             future.result()
 
+    @staticmethod
+    def _get_base64_svg(svg_filename):
+        svg_content = "data:image/svg+xml;base64,"
+        with open(svg_filename) as svg_file:
+            file_content = svg_file.read()
+            svg_content += base64.b64encode(file_content.encode()).decode()
+        return svg_content
+
+    def _embed_svg_images(self, html_dir):
+        p = Path(html_dir)
+        html_files = p.glob(f"**/*.html")
+        files_to_remove = set()
```

Comment:
is it common for doc to share same file across many html pages? i.e. are we increasing size by a lot by instead embedding image into every page?

Reply:
So far we get a single SVG per single Class doc page.
I haven't seen an example of generated shared images (non-shared non-generated are in the SVG_TO_KEEP list).